how-to/integrate-with-sso/integrate-with-auth0/public/manifest.fin.json

```diff
@@ -0,0 +1,45 @@
+{
+	"devtools_port": 9090,
+	"licenseKey": "openfin-demo-license-key",
+	"runtime": {
+		"arguments": "--v=1 --inspect",
+		"version": "28.106.72.17"
```

Comment:
Hopefully updating against main (which is now workspace 11) will mean this gets updated automatically. Otherwise it will need a runtime update and mentions of v10.0.0 will need to become v11.0.0